RELEASE.md

```diff
@@ -40,11 +40,8 @@ While the Travis build is ready for that, releasing locally requires the followi
     * `gpg --keyserver http://keyserver.ubuntu.com:11371 --send-keys {pub key ID}`
     * you can also use Web UI and upload plain test key that you can obtain via
       * `gpg --armor --export {your email used for the keys}`
-
-Run the following command to upload to Nexus. Note that it might be necessary to close and release via Nexus UI.
-```bash
-./gradlew upload -Psigning.keyId=<id> -Psigning.password=<pass> -Psigning.secretKeyRingFile=<home/user>/.gnupg/secring.gpg -PossrhUsername=<name> -PossrhPassword=<pas>
-```
+ * Run gradle upload to Nexus. Note that it might be necessary to close and release via Nexus UI.
+    * `./gradlew upload -Psigning.keyId=<key ID from gpg --list-secret-keys> -Psigning.password=<password you used to encrypt keys via gpg --gen-key> -Psigning.secretKeyRingFile=<home/user>/.gnupg/secring.gpg -PossrhUsername=<name> -PossrhPassword=<pas>`
```

Comment:
* `./gradlew upload` if you configured a `$HOME/.gradle/gradle.properties`;
* or `./gradlew upload -Psigning.keyId=<key ID from gpg --list-secret-keys> -Psigning.password=<password you used to encrypt keys via gpg --gen-key> -Psigning.secretKeyRingFile=<home/user>/.gnupg/secring.gpg -PossrhUsername=<name> -PossrhPassword=<pas>